app/views/imports/new.html.haml

```diff
@@ -6,16 +6,66 @@
       formComplete = $requiredFields.filter(-> @value == '').length == 0
       $submit.removeAttr 'disabled' if formComplete
 
-%h1 Create Import
+%h1 Create A New Import
 
-= form_for import do |f|
-  .form-group
-    = f.label :file_identifier, "Upload a CSV"
-    = f.file_field :file_identifier, required: true
-  .form-group
-    = f.label :source_id
-    = f.collection_select(:source_id, Source.all, :id, :name, {}, { class: 'form-control' })
-  .form-group
-    = f.label :description
-    = f.text_area :description, { class: 'form-control', required: true }
-  = f.submit 'Create', { class: 'btn btn-primary' }
+%ul{ class: %w[nav nav-tabs], role: 'tablist' }
+  %li{ class: 'nav-item'}
+    %a{ class: 'nav-link', href: '#help', data: { toggle: 'tab' }} Help
+  %li{ class: 'nav-item'}
+    %a{ class: %w[nav-link active], href: '#new', data: { toggle: 'tab' }} New
```

Comment:
I think this could be simplified - my rule of thumb is to use the class and id shorthand where possible:

```
%ul.nav.nav-tabs{ role: 'tablist' }
  %li.nav-item= link_to 'Help', '#help', class: 'nav-link', data: { toggle: 'tab' }
  %li.nav-item= link_to 'New', '#new', class: 'nav-link active', data: { toggle: 'tab' }
```